pom.xml

```diff
@@ -276,6 +298,33 @@ limitations under the License.
         <tag>HEAD</tag>
     </scm>
 
+    <developers>
+       <developer>
+         <name>Angus Davis</name>
+         <email>mngusdavis@google.com</email>
+         <organization>Google</organization>
```

Comment:
Also - I don't really mind being named, but the other option is a simple:
  <developers>
    <developer>
      <organization>Google Inc.</organization>
      <organizationUrl>http://www.google.com</organizationUrl>
    </developer>
  </developers>